config/packages/security.yaml

```
@@ -1,7 +1,7 @@
 parameters:
     sylius.security.admin_regex: "^/admin"
     sylius.security.api_regex: "^/api"
-    sylius.security.shop_regex: "^/(?!admin|api/.*|api$|media/.*)[^/]++"
+    sylius.security.shop_regex: "^/(?!admin|api|new-api/.*|api$|media/.*)[^/]++"
```

this regex is related to our old API (shop and admin) imo if it is necessary we shoud add new parameter for new API, instead of your approach.
```suggestion
    sylius.security.new_api_regex: "^/(?!new-api/.*|api$|media/.*)[^/]++"
```